### PR TITLE
Update macrel version to 1.2.0

### PR DESCRIPTION
**Pull request by Darcy220606**
## PR checklist

This PR is to opdate the `MACREL/CONTGS` version from 1.1.0 to 1.2.0. No parameters are changed, just the containers and md5Sums in `test.yml`. 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/nf-core/macrel/contigs/test.yml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
     - path: output/macrel/test/test.all_orfs.faa.gz
     - path: output/macrel/test/test.prediction.gz
     - path: output/macrel/test/test.smorfs.faa.gz
-      md5sum: 79704c6120c2f794518301af6f9b963d
+      md5sum: ba1d2339474bc393e6e4c7f28ef4fd46
     - path: output/macrel/test/test_log.txt
-      md5sum: 6fdba143dce759597eb9f80e5d968729
     - path: output/macrel/versions.yml
-      md5sum: be8bf0d0647751c635c3736655f29f85
+      contains: ["macrel"]
```
Comment (jfy133) [inline tests/modules/nf-core/macrel/contigs/test.yml:15]: That doesn't look right - the version should be just numbers: 1.2.0 in this case?

Normally we just check for the presence of this  file, not the contents

**Review by jfy133 — CHANGES_REQUESTED** [review]

File: tests/modules/nf-core/macrel/contigs/test.yml (hunk 1 above)
Comment (Darcy220606) [reply tests/modules/nf-core/macrel/contigs/test.yml:15]: ok in that case  i removed the md5sums and added `contains: ["macrel"]`

Comment (jfy133) [reply tests/modules/nf-core/macrel/contigs/test.yml:15]: You will? Or Have/ I don't see any change at th moment..

Comment (jfy133) [reply tests/modules/nf-core/macrel/contigs/test.yml:15]: You will? Or Have/ I don't see any change at th moment..

Comment (Darcy220606) [reply tests/modules/nf-core/macrel/contigs/test.yml]: Now, it didnt push for some reason

**Pull request merged**